vmngclient/api/dashboard_api.py

```diff
@@ -75,3 +75,79 @@ def get_control_statuses_count(self) -> DataSequence[Count]:
             status.status_list = DataSequence(Count, [create_dataclass(Count, item) for item in status.status_list])
 
         return statuses
+
+    def get_bfd_connectivity_count(self) -> DataSequence[Count]:
```

Comment:
I really do not understand the purpose  of the `Count` object. Why is it used for almost every endpoint? Can we have a simple description "Why?" and "What is it?"